iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -42,64 +42,64 @@
     <Compile Include="$(Common)\Logging.Common.cs">
       <Link>Common\Logging.Common.cs</Link>
     </Compile>
-    <Compile Include="$(Common)\service\AssertionFailedException.cs">
+    <Compile Include="$(Common)\iothub\service\AssertionFailedException.cs">
```

Comment:
These files are common between which projects? 
If you needed to specify "iothub" it means they are particular to service and should be moved together with the IoT Service SDK?